storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -1715,7 +1716,17 @@ static void log_status_local_parse(const char *s, log_status_t &log_status) {
   }
 }
 
-/** Read binaty log position and InnoDB LSN from p_s.log_status.
+/** Query server return UUIID_SHORT
+@param[in]   conn         mysql connection handle
+@return uuid_short */
+unsigned long long int get_uuid_short(MYSQL *connection) {
+  char *uuid = read_mysql_one_value(connection, "SELECT UUID_SHORT()");
+  unsigned long long uuid_short = strtoull(uuid, nullptr, 10);
+  free(uuid);
+  return (uuid_short);
+}
+
```

Comment:
this should be either in page tracking cc file or a common source file

Reply:
I have moved it to page tracking cc